CleverPush/CleverPush/Source/CPTopicsViewController.m

```diff
@@ -260,9 +268,8 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
     
     UISwitch* deselectSwitch = [[UISwitch alloc] init];
     CGSize switchSize = [deselectSwitch sizeThatFits:CGSizeZero];
-    deselectSwitch.frame = CGRectMake(tableView.bounds.size.width - switchSize.width - 5.0f, (44 - switchSize.height) / 2.0f, switchSize.width, switchSize.height);
+    deselectSwitch.frame = CGRectMake(tableView.bounds.size.width - (switchSize.width + 5.0), (44 - switchSize.height) / 2.0f, switchSize.width, switchSize.height);
```

Comment:
Can you please export the `44`, `5.0` and `2.0f` into constants / variables?

Reply:
Yes sure.